reviews/beautify_text.py

```diff
@@ -0,0 +1,19 @@
+import argparse
```

Comment:
encoding is not specified, something like this should be added at first or second line: # This Python file uses the following encoding: utf-8

Reply:
@janmalyy Encoding must specified only if you use non-ASCII characters in code. Which is the case, so good point!